pages/your_domain/microbial_biotechnology.md

```diff
@@ -12,27 +12,143 @@ In the event that no adequate Problem page exists for a problem that can be gene
+ 
+Due to the interdisciplinary nature of the field, data arising from studies in microbial biotechnology relate to both computational studies, such as modelling and simulation and the results of wet-lab based studies used for the construction and experimental characterisation of microbial systems. Given the breadth, scope and rapid development of the field of microbial biotechnology, this guide is by no means exhaustive.
 
-## Section 1 title
+## Design
  
 ### Description
 <!--- Sections within Domain pages (aside from "Introduction" at the start and "Relevant tools and resources " at the end) should focus on particular data management problems, which should be described in this first sub-section.
 For problems that are fully domain-specific, a detailed description is merited.
 For detailing the domain-specific challenges of a problem that is generic, please link to the corresponding generic Problem page before going into the domain-specific challenges. --->
+The design for a system in microbial biotechnology essentially involves two, interrelated exercises. (i) The identification of the biological entities/hosts that will be used to develop the product in question (ii) Identification of the genetic modifications/circuitry/constructs necessary to modify the host if appropriate. In some cases circuitry can be designed on its own, in a host independent manner, but at the time of writing this is still quite rare. Usually, the intended host context for a genetic system is specified beforehand. 
 
-### Considerations <!--- (optional) --->
+<!---### Considerations <!--- (optional) --->
 <!---  Direct and concise considerations, structured in bullet points and typically framed as questions RDMKit reader should ask themselves in order to arrive at the best solution among those listed below. One level of nesting of bullet points within considerations is fine, but more levels should be avoided. --->
 
 
 ### Solutions
 <!--- Detail, either in normal text or in bullet points, the domain-specific solutions to the problem. Do not merely list tools or resources, as they will be automatically listed in the bottom section, but you can and should mention tools and resources listed below if you detail their usage to solve the problem. --->
 
-<!--- ## Section 2 Title --->
+#### Biological hosts
 <!--- Add more sections as needed, with the same subsections as above. --->
-...
+For microbial biotechnology, bacteria and fungi make up the vast majority of hosts used in the biotechnology industry. These organisms are either sourced from a culture collection, another lab, developed in house or isolated from nature. Hosts will need to be described accordingly to record information about their source and their taxonomy. 
+
```

Comment:
The structure that needs to be followed is : Description, consideration and solutions.
Could you please add "considerations" section here. For guidance you can have a look at other domain pages